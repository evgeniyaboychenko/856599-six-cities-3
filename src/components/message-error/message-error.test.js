import React from 'react';
import renderer from 'react-test-renderer';
import MessageError from './message-error.jsx';
import {Provider} from "react-redux";
import configureStore from "redux-mock-store";


const mockStore = configureStore([]);
it(`should MessageError render correctly`, () => {
  const store = mockStore({
  });
  const tree = renderer.create(
      <Provider store={store}>
        <MessageError
          error = {`error`}
          onCloseClick = {() => {}}
        />
      </Provider>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
