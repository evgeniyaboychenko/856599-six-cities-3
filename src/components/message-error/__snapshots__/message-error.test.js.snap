// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should MessageError render correctly 1`] = `
<section
  style={
    Object {
      "backgroundColor": "#f5f5f5",
      "display": "block",
      "left": "50%",
      "marginLeft": "-250px",
      "position": "fixed",
      "top": "50%",
      "width": "500px",
      "zIndex": 30,
    }
  }
>
  <h2
    style={
      Object {
        "display": "flex",
        "textAlign": "center",
      }
    }
  >
    Ошибка 
    error
  </h2>
  <button
    aria-label="Закрыть модальное окно"
    onClick={[Function]}
    style={
      Object {
        "display": "flex",
        "margin": "auto",
      }
    }
    type="button"
  >
    Закрыть
  </button>
</section>
`;
